### PR TITLE
correct errors in English documents

### PR DESCRIPTION
**Pull request by JesseZhou-1**
Lots of changes are made, so please review carefully, thank you.

### CODE REVIEW

File: docs/UserGuide/Comparison/TSDB-Comparison.md [hunk 1]
```diff
@@ -48,12 +48,12 @@ We select 3 kinds of TSDBs here.
...
 Prometheus and Druid are also famous for time series data management. 
-However, Prometheus focuses on how to collect data, how to visualize data and how to alert warnings.
-Druid focuses on how to analyze data with OLAP workload. We omit them here.
+However, Prometheus focuses data collection, data visualization and alert warnings.
+Druid focuses on data analysis with OLAP workload. We omit them here.
  
 
 ## Comparison 
-We compare the above time series database from two aspects: the feature comparison and the performance
+The above time series database are comparedfrom two aspects: the feature comparison and the performance
```
Comment (HTHou) [inline docs/UserGuide/Comparison/TSDB-Comparison.md:56]:
```suggestion
The above time series databases are compared from two aspects: the feature comparison and the performance
```

File: docs/UserGuide/Comparison/TSDB-Comparison.md [hunk 2]
```diff
@@ -100,9 +100,9 @@ Legend:
 
 * SQL like: 
 
-  * IoTDB and InfluxDB supports SQL like language. In addition, The integration of IoTDB and Calcite is alomost done (a PR has been submitted), which means IoTDB will support Standard SQL.
+  * IoTDB and InfluxDB supports SQL like language. In addition, the integration of IoTDB and Calcite is alomost done (a PR has been submitted), which means IoTDB will support Standard SQL soon.
```
Comment (HTHou) [inline docs/UserGuide/Comparison/TSDB-Comparison.md:103]:
```suggestion
  * IoTDB and InfluxDB support SQL like language. In addition, the integration of IoTDB and Calcite is almost done (a PR has been submitted), which means IoTDB will support Standard SQL soon.
```

**Review by HTHou — COMMENTED** [review]
👍

**Review by qiaojialin — APPROVED** [review]

**Comment by qiaojialin** [comment]
Thank you Yuren!

**Comment by JesseZhou-1** [comment]
You are welcome : )

Jialin Qiao <notifications@github.com> 于2020年10月31日周六 下午3:41写道：

**Pull request merged**